src/main/java/core/basesyntax/machines/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.machines;
```

Comment:
```suggestion
package core.basesyntax.machine;
```
let's better use singular for package names

Reply:
There are few different machines. That is why I named that package in the plural. Could you explain please why do you think it's better to use a singular name here?